File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdatePlantSplitRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PlantsUpdatePlantSplitRequestItem {
    @JsonProperty("PlantCount")
    public Integer plantCount;
    @JsonProperty("SourcePlantLabel")
    public String sourcePlantLabel;
    @JsonProperty("SplitDate")
    public String splitDate;
    @JsonProperty("StrainLabel")
    public String strainLabel;
    @JsonProperty("TagLabel")
    public String tagLabel;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdatePlantSplitRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsUpdatePlantSplitRequestItem(
    @JsonProperty("plantCount")
    val plantCount: Int? = null,
    @JsonProperty("sourcePlantLabel")
    val sourcePlantLabel: String? = null,
    @JsonProperty("splitDate")
    val splitDate: String? = null,
    @JsonProperty("strainLabel")
    val strainLabel: String? = null,
    @JsonProperty("tagLabel")
    val tagLabel: String? = null
)
